docs/sources/flow/reference/components/loki.write.md

```diff
@@ -151,7 +151,7 @@ local Loki instance:
 ```river
 loki.write "local" {
     endpoint {
-        url = "loki:3100"
+        url = "http://loki:3100/loki/api/v1/push"
```

Comment:
Just a question... is this only `http` or should it be `https`?

Reply:
for local development, it is probably ok to use http (most people won't have certificates configured), for production it should be https, not sure if it's worth a comment